nntrainer/models/neuralnet.cpp

```diff
@@ -484,14 +484,9 @@ sharedConstTensors NeuralNetwork::inference(sharedConstTensors X,
 
   allocate(false);
 
-  try {
-    START_PROFILE(profile::NN_FORWARD);
-    forwarding(X, {}, false);
-    END_PROFILE(profile::NN_FORWARD);
-  } catch (...) {
-    ml_loge("Failed to inference Model");
-    return out;
-  }
+  START_PROFILE(profile::NN_FORWARD);
+  forwarding(X, {}, false);
+  END_PROFILE(profile::NN_FORWARD);
```

Comment:
Could you checkout line 482 as well? should we throw if `validateTensor` fails?

Reply:
done.